src/test/java/com/google/api/generator/gapic/dummy/BUILD.bazel

```diff
@@ -21,3 +23,39 @@ filegroup(
         "@junit_junit//jar",
     ],
 ) for test_name in TESTS]
+
+# JUnit runner binary, this is used to generate test output for updating goldens files.
+java_binary(
+    name = "junit_runner_binary",
+    srcs = [
+        ":dummy_files",
+        "//src/test/java/com/google/api/generator/test/framework:framework_files",
+    ],
+    data = ["//src/test/java/com/google/api/generator/gapic/dummy/goldens:goldens_files"],
+    jvm_flags = ["-Xmx512m"],
+    main_class = "com.google.api.generator.test.framework.SingleJUnitTestRunner",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//src/main/java/com/google/api/generator/engine/ast",
+        "//src/main/java/com/google/api/generator/engine/writer",
+        "//src/test/java/com/google/api/generator/test/framework",
+        "@io_github_java_diff_utils//jar",
+        "@junit_junit//jar",
+        "@org_hamcrest_hamcrest_core//jar",
+    ],
+)
+
+TEST_CLASS_NAME = "com.google.api.generator.gapic.dummy.FileDiffInfraDummyTest"
+
+# Run `bazel run src/test/java/com/google/api/generator/gapic/dummy:FileDiffInfraDummyTest.update`
+# to update goldens as expected generated code.
+update_golden(
+    name = "FileDiffInfraDummyTest.update",
```

Comment:
Thanks! My consideration is for keeping the consistency of the test name `FileDiffInfraDummyTest`, so that when we run `bazel run //src/test/..../dummy: FileDiffInfraDummyTest ` test fails then the command `bazel run //src/test/.../dummy:FileDiffInfraDummyTest.update` can help us update the goldens. But I'm open to modify this to `FileDiffInfraDummyTest_update` or `fileDiffInfraSummyTest_update` for matching lower_snake_case. Let me know your opinion on this.

Reply:
+1 to using `lower_snake_case`.

Reply:
Renamed to `FileDiffInfraDummyTest_update ` reserved the test name for consistency of test target name, and append `_update` for bazel target name convention. Thanks